reference/informative/informC06_interpretation_of_initial_values.rst

```diff
@@ -8,4 +8,117 @@
 
   .. container:: infospec
 
-    **TODO**
\ No newline at end of file
+    Best practice suggests that the :code:`initial_value` attribute is used to set the initial values only, rather than to simply set the value of a :code:`variable`.  
+    Initialising a variable is different from setting the value of a variable; the former is true only at the *beginning* of a simulation, whereas the latter is true *throughout* the simulation.
+    The option to use a variable reference for initialisation is provided in CellML 2.0 to enable easier migration from CellML 1.1 models, and may not be supported in future releases.
```

Comment:
Thanks! This is looking good but there's quite a lot going on here!
Maybe we could separate it into a few key points, in different paragraphs (possibly with headings?)

- semantics intial value true at t0 vs equation true always
- init used to set constants is not recommended but possible
- init can now be on connected var, allowing init vals to be set in a separate component (possibly imported)
  - init = var can be used to set init vals in separate component (possibly imported) but not recommended

Reply:
Thanks :) I don't quite understand how to put this bit into practice:

> init can now be on connected var, allowing init vals to be set in a separate component (possibly imported)
> init = var can be used to set init vals in separate component (possibly imported) but not recommended

So ... init = val in external component, no init through the rest of the connected set? So my "good" example is not really so good ... how should I change it?

Reply:
I'd just do an ODE with a component
```
component clever_stuff
    variable x
    variable y
    math:
        dx/dt = ...
        dy/dy = ...
component initialisation
    variable x initial_value=3
    variable y initial_value=7
```
?

Reply:
Yep, got it, thanks.